src/sentry/workflow_engine/processors/detector.py

```diff
@@ -55,6 +57,97 @@ def get_detector_by_event(event_data: WorkflowEventData) -> Detector:
     return detector
 
 
+def get_detectors_by_events_bulk(
+    event_list: list[GroupEvent],
```

Comment:
this would also need to support `Activity` models

Reply:
update: decided to not support Activity for now